### PR TITLE
[2.13.x] DDF-4391 DivisibleBy query does not work on multivalued fields

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
Fix bug that caused the DivisibleBy query not to work 
#### Who is reviewing it? 

@KalCramer 
@a-asaad 
#### Select relevant component teams: 
@codice/solr 
#### Ask 2 committers to review/merge the PR and tag them here.

@jrnorth
@pklinef

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/DDF-4391)
#### Checklist:
- [ ] Documentation Updated
- [ x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ryeats** [comment]
build now

File: catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java [hunk 1]
```diff
@@ -1283,6 +1286,52 @@ public void testPropertyIsInProximityTo() throws Exception {
     queryAndVerifyCount(0, getFilterBuilder().proximity(Core.TITLE, 1, "Mary little"), provider);
   }
 
+  @Test()
+  public void testPropertyIsDivibisibleBy() throws Exception {
```
Comment (jrnorth) [inline catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java]: Divisible

File: catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java [hunk 2]
```diff
@@ -1283,6 +1286,52 @@ public void testPropertyIsInProximityTo() throws Exception {
     queryAndVerifyCount(0, getFilterBuilder().proximity(Core.TITLE, 1, "Mary little"), provider);
   }
 
+  @Test()
```
Comment (jrnorth) [inline catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java]: Remove the ()

File: catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java [hunk 3]
```diff
@@ -1283,6 +1286,52 @@ public void testPropertyIsInProximityTo() throws Exception {
...
+  public void testPropertyIsDivibisibleBy() throws Exception {
+
+    deleteAll(provider);
+
+    String longField = "divisibleByCounter";
+    long longFieldValue = 6;
```
Comment (jrnorth) [inline catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java]: ✏️ Consider moving this next to the metacards below or using literals in `setAttribute()` to make it clear what the values are.

**Comment by ryeats** [comment]
build now

**Pull request merged**